apps/website/app/utils/llm/handler.ts

```diff
@@ -0,0 +1,119 @@
+import { NextRequest } from "next/server";
+import cors from "./cors";
+import {
+  CONTENT_TYPE_JSON,
+  CONTENT_TYPE_TEXT,
+  RequestBody,
+  LLMProviderConfig,
+} from "../../types/llm";
```

Comment:
~

Reply:
oh app is the root here I thought website is